frontend/composables/use-formatters.ts

```diff
@@ -26,99 +28,51 @@ export async function useFormatCurrency() {
+  const region = localeCode.length > 2 ? localeCode.substring(3) : "";
+  return Locales[(lang + region) as keyof typeof Locales] ?? Locales[lang as keyof typeof Locales] ?? Locales.enUS;
 }
 
 export function useLocaleTimeAgo(date: Date) {
-  const { t } = useI18n();
-
-  const I18N_MESSAGES: UseTimeAgoMessages<UseTimeAgoUnitNamesDefault> = {
-    justNow: t("components.global.date_time.just-now"),
-    past: n => (n.match(/\d/) ? t("components.global.date_time.ago", [n]) : n),
-    future: n => (n.match(/\d/) ? t("components.global.date_time.in", [n]) : n),
-    month: (n, past) =>
-      n === 1
-        ? past
-          ? t("components.global.date_time.last-month")
-          : t("components.global.date_time.next-month")
-        : `${n} ${t(`components.global.date_time.months`)}`,
-    year: (n, past) =>
-      n === 1
-        ? past
-          ? t("components.global.date_time.last-year")
-          : t("components.global.date_time.next-year")
-        : `${n} ${t(`components.global.date_time.years`)}`,
-    day: (n, past) =>
-      n === 1
-        ? past
-          ? t("components.global.date_time.yesterday")
-          : t("components.global.date_time.tomorrow")
-        : `${n} ${t(`components.global.date_time.days`)}`,
-    week: (n, past) =>
-      n === 1
-        ? past
-          ? t("components.global.date_time.last-week")
-          : t("components.global.date_time.next-week")
-        : `${n} ${t(`components.global.date_time.weeks`)}`,
-    hour: n => `${n} ${n === 1 ? t("components.global.date_time.hour") : t("components.global.date_time.hours")}`,
-    minute: n => `${n} ${n === 1 ? t("components.global.date_time.minute") : t("components.global.date_time.minutes")}`,
-    second: n => `${n} ${n === 1 ? t("components.global.date_time.second") : t("components.global.date_time.seconds")}`,
-    invalid: "",
-  };
-
-  return useTimeAgo(date, {
-    fullDateFormatter: (date: Date) => date.toLocaleDateString(),
-    messages: I18N_MESSAGES,
-  });
+  return formatDistance(date, new Date(), { locale: getLocale() });
 }
 
-export function fmtDate(value: string | Date, fmt: DateTimeFormat = "human"): string {
-  const months = [
-    "January",
-    "February",
-    "March",
-    "April",
-    "May",
-    "June",
-    "July",
-    "August",
-    "September",
-    "October",
-    "November",
-    "December",
-  ];
-
+export function fmtDate(value: string | Date, fmt: DateTimeFormat = "human", type: DateTimeType = "date"): string {
   const dt = typeof value === "string" ? new Date(value) : value;
-  if (!dt) {
+
+  if (!dt || !validDate(dt)) {
     return "";
   }
 
-  if (!validDate(dt)) {
-    return "";
+  if (fmt === "relative") {
+    return useLocaleTimeAgo(dt);
+  }
+
+  if (type === "time") {
+    return format(dt, "p", { locale: getLocale() });
   }
 
+  let formatStr = "";
+
   switch (fmt) {
-    case "relative":
-      return useLocaleTimeAgo(dt).value + useDateFormat(dt, " (YYYY-MM-DD)").value;
```

Comment:
Missing the (YYYY-MM-DD) localized equivalent now. I'll set the PR to draft until I fix that.

Reply:
Added short date in local format.